docs/unified-test-documentation/dasharo-security/208-reset-to-defaults.md

```diff
@@ -0,0 +1,49 @@
+# Dasharo Security: Reset to Defaults option
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+## RTD001.001 Reset to Defaults option
+
+**Test description**
+
+This test aims to verify that the `F9` key feature in Dasharo System Features
```

Comment:
I think that this section should be rewritten. The clue is to reset to defaults all Dasharo Features settings, no matter how they have been visualized. 

So that mention about each sub-menu is redundant.

Reply:
Updated